db/migrate/20241023080620_create_users.rb

```diff
@@ -0,0 +1,23 @@
+class CreateUsers < ActiveRecord::Migration[7.2]
+  def up
+    create_table :users do |t|
+      t.string :username
+      t.string :email
+      t.integer :role
+      t.string :password, null: false
+      t.string :phone
+
+      t.timestamps
+    end
+    add_index :users, :email, unique: true
+    add_index :users, :username, unique: true
+    add_index :users, :role
```

Comment:
why we are applying indexing on role column? as our role column has a very limited number of distinct values.

Reply:
Okay bhai. Removing it. Thank you!